cmsdev/internal/lib/common/cmd.go

```diff
@@ -191,7 +221,7 @@ func RunPathWithEnv(cmdEnv map[string]string, cmdPath string, cmdArgs ...string)
 	if err != nil {
 		return
 	}
-	err = cmdResult.Run()
+	err = cmdResult.RunWithRetry()
```

Comment:
RunWithPathEnv is used by non-CLI commands, so it should not be using the retry code.